crates/dojo-lang/src/compiler.rs

```diff
@@ -57,12 +60,21 @@ pub struct DojoCompiler;
 #[derive(Debug, Default, Serialize, Deserialize)]
 #[serde(rename_all = "kebab-case")]
 pub struct Props {
+    pub allowed_libfuncs_deny: bool,
+    pub allowed_libfuncs_list: Option<SerdeListSelector>,
```

Comment:
```suggestion
```
don't use them for now, as the target dojo is not profile specific anyway, so we can't have different configurations based on if it's slot or sepolia for instance.